manifests/kustomization.yaml

```diff
@@ -21,4 +21,4 @@ kind: Kustomization
 images:
 - name: cockroachdb/cockroach-operator
   newName: quay.io/alinalion/cockroach-operator
-  newTag: v1.0.0-alpha.3
+  newTag: v1.0.5-alpha.3
```

Comment:
Do we have to have this hardcoded?  I would rather pass in a version from Bazel.  That way we do not have to change files when we cut a release.

Reply:
I will open an issue with this